jax/experimental/compilation_cache/file_system_cache.py

```diff
@@ -37,6 +39,36 @@ def put(self, key: str, value: bytes):
     """Adds new cache entry, possibly evicting older entries."""
     if not key:
       raise ValueError("key cannot be empty")
-      #TODO(colemanliyah):implement eviction
-    with open(os.path.join(self._path, key), "wb") as file:
+
+    self._evict_entries_if_necessary(key, value)
+
+    self._create_a_file(self._path, key, value)
+
+  def _evict_entries_if_necessary(self, key:str, value:bytes):
+
```

Comment:
Remove blank line. Where to use blanks lines is generally a subjective stylistic preference, but it's pretty standard to not put a blank line at the beginning of a function.